tests/gce_virtual_machine_test.py

```diff
@@ -44,5 +47,24 @@ def testPreemptibleVMFlag(self):
       self.assertEquals(issue_command.call_count, 1)
       self.assertIn('--preemptible', issue_command.call_args[0][0])
 
+  def testImageProjectFlag(self):
+    """Tests that custom image_project flag is supported."""
+    with mock.patch(vm_util.__name__ + '.IssueCommand') as issue_command, \
```

Comment:
Now that this lengthy ```with``` appears twice, it'll be good to separate it into a helper or the setUp method for reuse. The name of the class is also no longer accurate.